pages/jobs.tsx

```diff
@@ -0,0 +1,52 @@
+import type { NextPage } from "next";
+import { getAllJobs } from "../lib/peopleHR";
+import type { JobPost } from "../lib/peopleHR";
+import styles from "styles/Jobs.module.scss";
+import Link from "next/link";
+
+interface JobPostWithSlug {
+    job: JobPost;
+    slug: string;
+}
```

Comment:
To be refactored with a new Job type that gets exported from `peopleHR.ts`. Best to manage data processing & slug generation in the `lib` file.